examples/example_02_matlab_function/output.txt

```diff
@@ -0,0 +1 @@
+0.000000
```

Comment:
Does this need to be part of version control?

Reply:
no. It should not appear on the next comit